packages/DropdownMenu/src/components/LinkItem/LinkItem.js

```diff
@@ -33,15 +34,18 @@ const LinkItem = props => {
     ...moreProps,
   };
 
+  let externalLinkIcon = null;
   if (isExternal) {
     linkItemProps["aria-label"] = I18n.t("dropdownMenu.isExternal", { link: children });
     linkItemProps.target = "_blank";
     linkItemProps.rel = "noopener noreferrer";
+    externalLinkIcon = <NewTabIcon size="10px" />;
```

Comment:
Perhaps we could leverage the `Button.Link` component 
```
if (isExternal) {
    linkItemProps.a11yText = I18n.t("dropdownMenu.isExternal", { link: children });
    linkItemProps.shouldOpenNewTab = true;
    linkItemProps.suffixIcon = null;
  }

  return (
    <Button.Link css={linkItemStyles} {...linkItemProps}>
      {children}
    </Button.Link>
  );
```

Reply:
Hmm I don't think that's necessary, I'm not experiencing any focus ring style issues as-is.

Reply:
Ok np I will address the keyboard navigation focus bug (when first item is a link) in a follow up pr.